code/datums/components/surgery_initiator.dm

```diff
@@ -69,7 +69,7 @@
 					continue
 			else if(carbontarget && surgeryloop_two.requires_bodypart) //mob with no limb in surgery zone when we need a limb
 				continue
-			if(surgeryloop_two.lying_required && (livingtarget.mobility_flags & MOBILITY_STAND))
+			if(surgeryloop_two.lying_required && livingtarget.body_position != LYING_DOWN)
```

Comment:
Not lying down and standing up are interchangeable to a certain degree. At least until more body positions are added.
In this case I went with not-lying-down because that's a bit more safe of a check in case things change.